keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -299,23 +305,21 @@ the API proposed by this KEP aims to
   clusters under management.
 
 ### Terminology
+- **Cluster Inventory**: A conceptual term referring to a collection of clusters.
 
-- **Cluster Manager**: An entity that creates the ClusterProfile API
-  object per member cluster, and keeps their status up-to-date. Each
-  cluster manager SHOULD be identified with a unique name. Each cluster
-  profile resource SHOULD be managed by only one cluster manager. Examples
-  of cluster manager are projects like OCM, Karmada, Clusternet or Azure
-  fleet manager.
+- **Member Cluster**: A kubernetes cluster that is part of a cluster inventory.
+
+- **Cluster Manager**: An entity that creates the ClusterProfile API object per member cluster,
+  and keeps their status up-to-date. Each cluster manager MUST be identified with a unique name.  
+  Each ClusterProfile resource SHOULD be managed by only one cluster manager. A cluster manager SHOULD 
+  have sufficient permission to access the member cluster to fetch the information so it can update the status
+  of the ClusterProfile API resource. Examples of cluster manager are projects like [OCM](https://open-cluster-management.io/),
+  [Clusternet](https://clusternet.io/) or [Kubernetes Fleet Manager](https://github.com/Azure/fleet)
```

Comment:
ditto
Karamda should be here.  Here is the website: https://karmada.io/

Reply:
agree